docs/workflows/best-practices-organization.md

```diff
@@ -71,6 +71,10 @@ There are also disadvantages. Not all codebases are big or important enough to d
 ### A platform team
 Platform teams *enable* non-platform teams (aka delivery/feature/stream-aligned teams) to deliver value. They do not typically work on the same codebase as non-platform teams, although they could, especially if multiple non-platform teams share a codebase (the [tech lead](#a-hands-on-tech-lead) could be in that platform team in that case). They can however help teams identify and make technical improvements by providing tooling, e.g. performance or code quality measurement tooling. Next, platform teams can play a big role in improving the developer experience (DevEx), e.g. by speeding up pipelines. Also, they can provide expertise in areas like databases, development pipelines and container orchestration.
 
+### A dedicated software quality team
+A dedicated software quality team enables development teams to build high-quality software. They have knowledge of all aspects of software quality, assist teams with improving the quality of their software and share best practices across teams. They can help set standards and report on compliance.
+Advantages are clarity and centralization. Teams know where to go to get assistance or advice on the topic. Management knows where to get insights in organizational performance in this area. The flip side of these advantages is that having a dedicated team may cause people outside the team to feel less responsibility to pick up the topic of software quality and look at the team to 'fix things'. Since the team has no direct influence on software and is enabling in nature, that will not happen.
```

Comment:
IND sort of has this setup of a dedicated swq team, 2 people working with 6-10 teams across 6 different technologies. This works with some teams and not with others, with the distinguishing element being a tech lead or champion within the team who is willing to help coach or guide the rest of the team. This instance of the swq still requires that hands-on tech lead, and their main advantage is alleviating the coordination, (Sigrid) administration and reporting burden from the tech leads.

All this to say:
```suggestion
Advantages are clarity and centralization. Teams know where to go to get assistance or advice on the topic. Management knows where to get insights in organizational performance in this area. The flip side of these advantages is that having a dedicated team may cause people outside the team to feel less responsibility to pick up the topic of software quality and look at the team to 'fix things'. Since the team has no direct influence on software and is enabling in nature, that will not happen. Successful software quality teams often work with hands-on tech leads to instil a sense of ownership within the teams for quality standards.
```